<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_1">
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-10</y>
          <width>381</width>
          <height>541</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="check_your_work">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>490</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Check your work</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>2</x>
           <y>490</y>
           <width>91</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>391</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>28</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Get Started Now</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>240</y>
           <width>381</width>
           <height>68</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>181</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>password</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="lineEdit_password"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>125</y>
           <width>381</width>
           <height>66</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_username"/>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>340</y>
           <width>63</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>330</y>
           <width>381</width>
           <height>66</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Email address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Email"/>
          </item>
         </layout>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>410</y>
           <width>191</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>I agree with the policy</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_signup">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>440</y>
           <width>93</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>signup</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_pic">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>-10</y>
          <width>401</width>
          <height>571</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>170</y>
          <width>291</width>
          <height>231</height>
         </rect>
        </property>
        <property name="text">
         <string>THIS IS FOR SHOWING MAPS</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>go to original page</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>99</y>
          <width>301</width>
          <height>231</height>
         </rect>
        </property>
        <property name="text">
         <string>THIS IS FOR MESSAGES ONLY</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumaps">
    <property name="title">
     <string>maps</string>
    </property>
    <addaction name="actionNavigate"/>
   </widget>
   <addaction name="menumaps"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNavigate">
   <property name="text">
    <string>Navigate</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit_username</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>check_your_work</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>pushButton_4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
